wagtail/snippets/templates/wagtailsnippets/snippets/index_results.html

```diff
@@ -1,10 +1 @@
 {% extends "wagtailadmin/generic/index_results.html" %}
-{% load i18n %}
-
-{% block no_results_message %}
-    {% if is_searching or is_filtering %}
-        <p role="alert">{% blocktrans trimmed with snippet_type_name_plural=model_opts.verbose_name_plural %}Sorry, no {{ snippet_type_name_plural }} match your query{% endblocktrans %}</p>
-    {% else %}
-        <p class="no-results-message">{% blocktrans trimmed with snippet_type_name_plural=model_opts.verbose_name_plural wagtailsnippets_create_url=add_url %}No {{ snippet_type_name_plural }} have been created. Why not <a href="{{ wagtailsnippets_create_url }}">add one</a>?{% endblocktrans %}</p>
-    {% endif %}
-{% endblock %}
```

Comment:
I opted to remove this, but we can also move this to the generic templates if desired

Reply:
I'd prefer to see this reinstated in the generic templates - or at least we should have better justification for removing it than "it saves us having to fix #11393" :-). I have no strong feelings about the message personally, but I think it's better that we don't casually remove things that people might be accustomed to, or have built workflows around.

Besides that, these changes look good to me!